kernel/pci/pci.c

```diff
@@ -11,130 +11,130 @@ pci_device_t pci_devices[MAX_PCI_DEVICES] = { 0 };
 void
 enable_bus_mastering(uint32_t address)
 {
-	uint16_t val = read16(address + PCI_COMMAND);
-	val |= PCI_COMMAND_MASTER | PCI_COMMAND_IO | PCI_COMMAND_MEMORY;
-	write16(address + PCI_COMMAND, val);
+  uint16_t val = read16(address + PCI_COMMAND);
+  val |= PCI_COMMAND_MASTER | PCI_COMMAND_IO | PCI_COMMAND_MEMORY;
+  write16(address + PCI_COMMAND, val);
 }
 
 uint8_t
 get_bar_type(uint32_t base, uint8_t number)
 {
-	return read32(base + PCI_BAR(number)) & 1;
+  return read32(base + PCI_BAR(number)) & 1;
 }
 
 // Refer to page 204 in PCI 2.2 Spec 
 // https://www.ics.uci.edu/~harris/ics216/pci/PCI_22.pdf
 uint32_t
 get_bar_size(uint32_t base, uint8_t number)
 {
-	// read value before to restore later
-	uint32_t bar_addr = base + PCI_BAR(number);
-	uint32_t before = read32(bar_addr);
-	// write all 1s to get encoded size information
-	write32(bar_addr, 0xFFFFFFFF);
-	// read encoded size information
-	uint32_t encoded_size = read32(bar_addr);
-	// restore old value
-	write32(bar_addr, before);
-
-	// I/O space base address register | memory space base address register
-	uint32_t bit_mask = (before & 1) ? 0xFFFFFFFC : 0xFFFFFFF0;
-	uint32_t decoded_size = (~(encoded_size & bit_mask)) + 1;
-
-	return decoded_size;
+  // read value before to restore later
+  uint32_t bar_addr = base + PCI_BAR(number);
+  uint32_t before = read32(bar_addr);
+  // write all 1s to get encoded size information
+  write32(bar_addr, 0xFFFFFFFF);
+  // read encoded size information
+  uint32_t encoded_size = read32(bar_addr);
+  // restore old value
+  write32(bar_addr, before);
+
+  // I/O space base address register | memory space base address register
+  uint32_t bit_mask = (before & 1) ? 0xFFFFFFFC : 0xFFFFFFF0;
+  uint32_t decoded_size = (~(encoded_size & bit_mask)) + 1;
+
+  return decoded_size;
 }
 
 // Currently there are no additional pci devices, thus there is no memory management
 uint32_t
 alloc_pci_memory(pci_device_t* device, uint8_t bar)
 {
-	uint32_t size = get_bar_size(device->config_base, bar);
-	uint8_t type = get_bar_type(device->config_base, bar);
-	uint32_t address = 0;
-	if(type)
-		{
-			// io space
-			address = 0x43000000;
-		}
-	else
-		{
-			// memory space			
-			address = 0x50000000;
-		}
-	write32(device->config_base + PCI_BAR(bar), address);
-	return address;
+  uint32_t size = get_bar_size(device->config_base, bar);
+  uint8_t type = get_bar_type(device->config_base, bar);
+  uint32_t address = 0;
+  if(type)
+    {
+      // io space
+      address = 0x43000000;
+    }
+  else
+    {
+      // memory space     
+      address = 0x50000000;
```

Comment:
kann man deutlich kuerzer fassen, wen man die Klammern weglaesst

Reply:
technically yes, I just thought that one might extend that later and add some kind of memory management, thus leaving the brackets for easier extendability